cypress/integration/search_test.js

```diff
@@ -0,0 +1,80 @@
+describe("Search Test", function(){
+    beforeEach(function(){
+        cy.visit("/");
+    });
+
+    it("can be accessed from navigation bar", function(){
+        cy.searchKeyword("Ma*");
+        cy.url().should("include","/search");
+
+        cy.get(".results > form").children().first().click();
+        cy.url().should("include","/report");
+    });
+
+    it("can search a single term", function(){
+        cy.searchKeyword("ABRA");
+        cy.url().should("include","/search");
+
+        cy.get(".results > form").children().first().click();
+        cy.url().should("include","/report");
+        cy.contains("Symbol").next().should("have.text","ABRA");
+    });
+
+    it("can search two terms with OR", function(){
+        cy.searchKeyword("CDPK1 OR CDPK4{enter}");
+        cy.url().should("include","/search");
+
+        cy.get(".results > form").children().first().within(()=>{
+            cy.contains("Symbol").next().should("have.text","CDPK1");
+        });
+
+        cy.get(".results > form").children().last().within(()=>{
+            cy.contains("Symbol").next().should("have.text","CDPK4");
+        });
+    });
+
+    it("can search a phrase with quotation marks", function(){
+        cy.searchKeyword('"DNA binding"');        
+        cy.url().should("include","/search");
+
+        cy.get(".results > form").children().first().click();
+        cy.url().should("include","/report");
+        cy.contains("Name").next().should("have.text","DNA binding");
+    });
+
+    it("can search for partial matches", function(){
+        cy.searchKeyword('MAL*');        
+        cy.url().should("include","/search");
+
+        cy.get(".results > form").children().first().click();
+        cy.url().should("include","/report");
+        cy.get('.report-table-body').within(() => {
+            cy.get(".report-table-cell").filter(':contains("MAL")');
```

Comment:
Are you sure this will fail if there's no MAL? Maybe you have to add `.its('length').should("be.gt", 0);`